src/components/subscribe/subscribe.ts

```diff
@@ -8,8 +8,9 @@
  *   Subscribe component elements
  * - The form can be hidden by pressing the `Escape` key
  */
-export const initSubscribe = (containerEl: HTMLElement) => {
+export const createSubscribe = (containerEl: HTMLElement) => {
   const SHOW_FORM_CLASS = 'activate-form';
+  const DESTROYED_STATE_CLASS = 'is-destroyed';
```

Comment:
I think part of the reason I was wondering if `:only-child` works or not is that I don't really like this class name.

The class name relates to the JavaScript... the component was destroyed. But the HTML elements and their styles remain, so `is-destroyed` feels strange there. Is it really supposed to be `is-inactive`? Or should it be `is-enhanced` when `is-destroyed` is _not_ true?

Something's sticking in my craw about it right now. Of course, if `:only-child` worked for that selector it might be a completely moot point!

Reply:
@tylersticka Fair points!

Since the `is-destroyed` CSS state class has a 1:1 relationship with the `destroy()` method, does that mean you are suggesting something other than "destroy" for the method name as well? 🔍 👀

Reply:
Update: [It's moot](https://github.com/cloudfour/cloudfour.com-patterns/pull/1882#discussion_r910309872)! 🕺🏽